engine/graphics/src/opengl/graphics_opengl_defines.h

```diff
@@ -154,63 +154,63 @@
 
 #if defined(GL_RGB32F_EXT)
 #define DMGRAPHICS_TEXTURE_FORMAT_RGB32F                    (GL_RGB32F_EXT)
-#elif defined(GL_RGB32F) && !defined (__EMSCRIPTEN__)
+#elif defined(GL_RGB32F)
```

Comment:
These are all core now on WebGL2

Reply:
And for WebGL1 we have the fallback to the "correct" float types for that context.